pulsar-package-management/core/pom.xml

```diff
@@ -48,4 +48,16 @@
             <artifactId>commons-lang3</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <configuration>
+                    <excludeFilterFile>${basedir}/src/main/resources/findbugsExclude.xml</excludeFilterFile>
```

Comment:
```suggestion
                    <excludeFilterFile>${basedir}/src/main/resources/findbugs-exclude.xml</excludeFilterFile>
```

Reply:
We are using this file for the exclude filter: https://github.com/apache/pulsar/pull/8784/files#diff-2ece31f925717628e3a3b606d38970a684b39b9e37f15d90fde5cea381420b02